edx-platform/pearson-theme/lms/templates/index.html

```diff
@@ -8,6 +8,10 @@
 from openedx.core.djangolib.markup import HTML, Text
 from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
 %>
+<%
+search_courses_home = configuration_helpers.get_value('search_courses_home', '')
+quick_filters_home = configuration_helpers.get_value('quick_filters_home', '')
```

Comment:
If these settings are array or dict or any other type, make sure that default type value is according to the excepted type value. @amalbas